docs-2.0/1.introduction/2.data-model.md

```diff
@@ -0,0 +1,47 @@
+# Data model
+
+This document gives some introduction to the Nebula Graph data model. A data model is a model that organizes data and specify how they are related to one another.
+
+## Data structures
+
+Nebula Graph data model uses five data structures to store data. They are vertices, edges, properties, tags, and edge types.
+
+- **Vertices**: Vertices are used to store entities.
+  - In Nebula Graph, vertices are identified with vertices identifiers (i.e. `VID`). The `VID` must be unique in the same graph space.
+  - A vertex must have at least one tag.
+- **Edges**: Edges are used to connect vertices. An edge is a connection or behavior between two vertices.
+  - An edge is identified uniquely with a source vertex, an edge type, a rank value, and a destination vertex.
+  - Edges are directed. `->` identifies the directions of edges. Edges can be traversed in either direction.
+  - An edge must have one and only one edge type.
+  - The rank value is an immutable user-assigned 64-bit signed integer. It identifies the edges with the same edge type between two vertices. Edges with higher rank values come first. The default rank value is zero.
+- **Properties**: Properties are key-value pairs. Both vertices and edges are containers for properties.
+- **Tags**: Tags are used to categorize vertices. Vertices that have the same tag are collected into a group.
+- **Edge types**: Edge types are used to categorize edges. Edges that have the same edge type are collected into a group.
+
+## Directed property graph
+
+Nebula Graph uses a directed property graph to keep data. A directed property graph has a set of vertices connected by edges. And the edges have directions. A directed property graph is represented as:
+
+**G = < V, E, P<sub>V</sub>, P<sub>E</sub> >**
+
+- **V** is a set of vertices.
+- **E** is a set of directed edges.
+- **P<sub>V</sub>** is the property on vertices.
```

Comment:
Is it OK that we use html tags here? Or maybe just use PV?

Reply:
I think it's ok here

Reply:
it's ok